src/deepsparse/transformers/utils/helpers.py

```diff
@@ -211,3 +212,61 @@ def create_causal_mask(
     causal_mask[:, :, :, :num_zeros] = 0
 
     return causal_mask
+
+
+def update_context_to_single_stream(**kwargs) -> Context:
```

Comment:
we don't want to create a context unless one already exists

Reply:
Updating the code, but would like to understand why is this the case?